addons/hr_expense/static/src/mixins/expense_dashboard.js

```diff
@@ -0,0 +1,36 @@
+/** @odoo-module */
+
+import { useService } from '@web/core/utils/hooks';
+import session from 'web.session';
+
+const { onWillStart, useState } = owl;
+
+
+export const ExpenseDashboardMixin = {
+    setup() {
+        this._super();
+        this.orm = useService('orm');
+
+        this.state = useState({
+            expenses: {}
+        });
+
+        onWillStart(async () => {
+            const expense_states = await this.orm.call("hr.expense", 'get_expense_dashboard', []);
+            this.state.expenses = expense_states;
+        });
+    },
+
+    render_monetary_field(value, currency_id) {
```

Comment:
Would recommend sticking to lowerCamelCase in js
```suggestion
    renderMonetaryField(value, currency_id) {
```